src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -182,7 +182,8 @@ private async Task StartDashboardAsync(CancellationToken cancellationToken = def
         await WaitForHttpSuccessOrThrow(dashboardUrl, TimeSpan.FromSeconds(DashboardWaitTimeInSeconds), cancellationToken).ConfigureAwait(false);
     }
 
-    private const int DashboardWaitTimeInSeconds = 10;
+    // Allow extra time when building locally. Dashboard can take longer to start up because of dev binaries, debugger, etc
```

Comment:
```suggestion
    // Dashboard should always launch successfully within this time.
    // Also, gives necessary time when developing the dashboard. Dashboard can take longer to start up because of dev binaries, debugger, break points, etc
```